modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceDBPartitionTest.java

```diff
@@ -400,6 +402,8 @@ public void testCopyDBPartitionCompany() throws Exception {
 
 			_assertCompanyConfiguration(copiedCompanyId, configuration);
 
+			_addCopyDBPartitionCompanyCache(copiedCompanyId);
```

Comment:
@marianoalvarosaiz What's the status should I expect after this adding? You only add stuff, but did not assert